.github/actions/test-linux/action.yml

```diff
@@ -43,6 +43,7 @@ runs:
         export TEST_PHP_JUNIT=junit.out.xml
         export STACK_LIMIT_DEFAULTS_CHECK=1
         sapi/cli/php run-tests.php -P -q ${{ inputs.runTestsParameters }} \
+          ${{ inputs.jitType == 'disable' && '' || '--repeat 2' }}
```

Comment:
The issue with combining `--repeat` with ASAN is that `--repeat` disabled certain tests (e.g. all tests with a `CLEAN` section). So this would essentially stop testing all DB extensions on ASAN, amongst other things.